src/nginxconfig/generators/to_conf.js

```diff
@@ -97,7 +97,7 @@ const recurse = (entriesOrObject, depth) => {
         // Work through each item in the array
         val.forEach(subVal => {
             const val = subVal.toString();
-            retVal += indent + (item[0] + keyValIndent + val).trim() + (item[0].startsWith('#') ? '' : ';') + '\n';
+            retVal += indent + (item[0] + keyValIndent + val).trim() + (item[0].startsWith('#') ? '' : endLine) + '\n';
```

Comment:
```suggestion
            retVal += indent + (item[0] + keyValIndent + val).trim() + (item[0].startsWith('#') ? '' : ';') + '\n';
```
This change can be reverted